src/components/Modals/AddClientModal.jsx

```diff
@@ -33,7 +33,7 @@ import { FormSection } from '../Form';
 
 const renderWebId = (username) => {
   const oidcProvider = ENV.VITE_SOLID_IDENTITY_PROVIDER.split('//')[1];
-  const template = ['https://', `.${oidcProvider}profile/card#me`];
+  const template = ['https://', `.${oidcProvider}/profile/card#me`];
```

Comment:
We can actually make this fix a little better if we change both lines 35 and 36

We shouldn't be loading the oidcProvider from the ENV. We store the oidcProvider used during login in localstorage to fetch later. We can also construct these Urls using javascript's builtin `URL` object to avoid needing to deal with these dashes.

```suggestion
const renderWebId = (username) => {
  const baseUrl = new URL(localStorage.getItem('oidcProvider'));
  return new URL(`${username}/profile/card#me`, baseUrl);
}
```

This will technically return a webId in a different format than before, but it's the format that's used by the community solid server we're using as a dev server, so it shouldn't matter that much.

Reply:
@timbot1789 That changes the outcome url's structure. Is that ok?

Reply:
Yeah that'll be fine. It's the format used by our dev server. And if you want to use a different format, you can manually edit the field

Reply:
Cool, I made the change